CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 # CHANGELOG
 
+## 2.10.0 (2024-01-26)
+
+MLflow 2.10.0 includes several major features and improvements
+
+Features:
+
+- [Tracking] Langchain autologging support (#10801, @serena-ruan)
+- [Models] Objects and Array support in model signature (#9936, @serena-ruan)
+- [Models] Support saving prompt templates for transformers (#10791, @daniellok-db)
+- [Scoring] Allow mlflow model serving to support direct dict inputs with "messages" key (#10742, @daniellok-db, @B-Step62)
+- [Models] Enhance `predict` API to serve for env validation purpose. (#10759, @B-Step62)
+- [Deployments] Gateway Streaming for OpenAI (#10765, @gabrielfu)
+- [UI] MLflow UI Update (#10864, @daniellok-db)
+- [Models] Add support for retrieving license files for transformers models (#10871, @BenWilson2)
+- [Deployments] Add rate limit to deployment api (#10779, @TomeHirata)
+- [Models] Automatically convert OpenAI input messages to LangChain chat messages for pyfunc predict (#10758, @dbczumar)
+- [Tracking] Add flush functionality to async logging (#10715, @chenmoneygithub)
+- [Deployments] Create `mlflow.deployments.openai` (#10473, @prithvikannan)
+- [Models] Support for OCR jars and minor refactor for defining wheel URLS (#10603, @C-K-Loan)
+- [Tracking] Feature/non interactive databricks login (#10623, @henxing)
+- [Server-infra] Add env variable to disable redirects (#10655, @daniellok-db)
+- [Artifacts] Proxy Multipart Upload for Azure Blob Storage (#10531, @gabrielfu)
+
+Bug fixes:
+
+- [Recipe] Fix recipe card display format (#10893, @WeichenXu123)
+- [Tracking] Fix an issue with an incorrect retry default timeout for urllib3 1.x (#10839, @BenWilson2)
+- [Windows] Replace Window shell-escaping of databricks run command with bash escaping (#10811, @wolpl)
+- [Java] [WIP] Fix ReplAwareDatasourceAttributeExtractor (#10729, @daniellok-db)
```

Comment:
oops! i forgot to remove [WIP] from this title
```suggestion
- [Java] Fix ReplAwareDatasourceAttributeExtractor (#10729, @daniellok-db)
```

Reply:
ooh thanks for the catch!